spec/factories/adoption_app.rb

```diff
@@ -24,6 +24,7 @@
     vet_info 'Dr. Spaceman Baltimore MD'
     max_hrs_alone 3
     is_ofage true
+    has_family_under_18 true
```

Comment:
if our default is nil then why give this a value of true in the factory?

Reply:
Should it be nil then in the factory? I am not familiar with the use of this and I just thought that I was just testing it for the purpose of seeing if it would work?

Reply:
@blai00 if you've done very little with RSpec testing, consider checking out https://leanpub.com/everydayrailsrspec